site/src/pages/docs/client-decorator.mdx

```diff
@@ -169,6 +169,56 @@ preview because <type://ContentPreviewingClient> is evaluated first for the HTTP
 For <type://RetryingClient>, please check out 
 [`RetryingClient` with logging](/docs/client-retry#retryingclient-with-logging).
 
+## Handling responses
+
+### Modifying response
+
+Modifying the responses can be achieved from within a `ClientBuilder.decorator()` either via using `HttpResponse.aggregate()`, `ClientRequestContext.mutateAdditionalRequestHeaders()` or via a wrapping of the `HttpResponse`, e.g. via `FilteredHttpResponse`:
```

Comment:
> `ClientRequestContext.mutateAdditionalRequestHeaders()`

This is for adding headers to the request so we can remove it.

> HttpResponse.aggregate()

Well aggregating the response in the decorator is not recommended so let's remove that too. 😄 

How about just:
```
We can modify the responses in a decorator via wrapping the <type://HttpResponse> with <type://FilteredHttpResponse>:
```

Reply:
Doh. Yes, mutateAdditionalRequestHeaders isn't meaningful for responses.

I wasn't sure what to put in the docs, but aiming for keeping it to recommended instead of having every possible solution makes a lot of sense. 

I wasn't familiar with `<type://>` either. I'll go over and fix it.